docs/Blog-Post.md

```diff
@@ -0,0 +1,276 @@
+# ---
+layout: blog
+title: Levanter — Towards Scaling Reproducible Foundational Models with Jax
+authors:
+    - name: David Hall
+      url: TODO
+    - name: Percy Liang
+      url: https://cs.stanford.edu/~pliang/
+display: False
+---
+> <div class="blog-tagline">
+    <strong> We introduce our projects
+        <a href="https://github.com/stanford-mercury/levanter" target="_blank">Haliax and Levanter</a>, our code
+        and infrastructure for training reproducible, legible foundation models models using Jax. We also XXX checkpoints.
+    </strong>
+> </div>
+
+
+## Introduction
+
+The growth of artificial intelligence and machine learning has brought about the need for scalable and reproducible models.
+To address this, at the [Center for Research on Foundation Models](https://crfm.stanford.edu), we have created two new tools — [Levanter and Haliax](https://github.com/stanford-crfm/levanter).
+They form a new code base for training foundational models with the promise of flexibility, modularity, efficiency,
+and scale, as well as strong guarantees about reproducibility.
+
+Levanter is a [Jax](https://github.com/google/jax)-based codebase for training foundation models that is designed to be
+flexible, modular, and accessible, while still being performant and scalable. In particular, Levanter is designed to be
+able to train models on a variety of different hardware, including GPUs, TPUs, and TPU pods. Levanter also offers strong
+guarantees about reproducibility, and we have released checkpoints for a number of models trained with Levanter.
+Haliax is a named tensor library that we developed to make it easier to write legible, composable code while
+still maintaining efficiency and scalability.
+
+Today, we're releasing the first version of Levanter and Haliax, along with several models trained with Levanter. We
+hope that these libraries will be useful to the community, and we look forward to seeing what people do with them.
+
+## The Landscape of Foundation Model Training Frameworks
+
+Numerous foundation model training frameworks exist in the community, each with its strengths and focus.
+For large language models (LLMs) (the focus of this release),
+the most well-known in the open source community is probably NVIDIA's PyTorch-based [Megatron-LM](https://github.com/NVIDIA/Megatron-LM),
+and its many derivatives, including EleutherAI's [GPT-NeoX](https://github.com/EleutherAI/gpt-neox) codebase. MosaicML
+has recently released [LLM Foundry](https://github.com/mosaicml/llm-foundry), which we used to train
+[BioMedLM](https://crfm.stanford.edu/2022/12/15/biomedlm.html) last autumn.
```

Comment:
Anything else?  HF / mistral?  metaseq?

Reply:
i had an old version that had a "too many to list" comment but I left it out of this draft. i added the ones you mentioned plus a couple others.